hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java

```diff
@@ -206,6 +206,7 @@
                long callTimeoutMillis,
                boolean deserialize,
                ServerConnectionManager connectionManager) {
+        super(context.logger);
```

Comment:
Also, I see that there are some places that uses `context.logger` for logging purposes. Can we also replace them with this?